playpen/dev-setup.sh

```diff
@@ -6,17 +6,31 @@ if [[ $EUID -eq 0 ]]; then
     exit 1
 fi
 
+read -p "What is your GitHub username? [$USER] " GITHUB_USERNAME
+if [ "$GITHUB_USERNAME" = "" ]; then
+    GITHUB_USERNAME="$USER"
+fi
+echo "Choosing $GITHUB_USERNAME as your GitHub username."
+
+read -p 'Where would you like your code checked out? [$HOME/devel] ' DEVEL_PATH
+if [ "$DEVEL_PATH" = "" ]; then
+    DEVEL_PATH="$HOME/devel"
+fi
+echo "Choosing $DEVEL_PATH as your development path."
```

Comment:
Add a prompt here for which repos to check out that defaults to the list below. This way, devs don't have to fork everything if they don't want to.